claim-check/code-samples/sample-4/bicep/main.bicep

```diff
@@ -29,6 +29,20 @@ resource eventHubDataOwnwerRole 'Microsoft.Authorization/roleDefinitions@2022-04
 
 /*** NEW RESOURCES ***/
 
+resource logAnalytics 'Microsoft.OperationalInsights/workspaces@2021-12-01-preview' = {
+  name: 'la-${namePrefix}'
+  location: location
+  properties: any({
+    retentionInDays: 30
+    features: {
+      searchVersion: 1
+    }
+    sku: {
+      name: 'PerGB2018'
+    }
+  })
```

Comment:
`any()`

Do you have more PR to fix this in?

Reply:
Sorry, it was a copy and paste. I didn’t notice until you pointed it out, and there’s no execution error. I’ll need to check each one individually.